source/isaaclab/isaaclab/actuators/actuator_cfg.py

```diff
@@ -33,13 +33,41 @@ class ActuatorBaseCfg:
     effort_limit: dict[str, float] | float | None = None
     """Force/Torque limit of the joints in the group. Defaults to None.
 
-    If None, the limit is set to the value specified in the USD joint prim.
+    This limit is used to clip the computed torque sent to the simulation. If None, the limit is set to the value
+    specified in the USD joint prim.
+
+    For ImplicitActuators this value will be collapsed with effort_limit_sim due to duplicating functionality. If both
+    are set the effort_limit_sim will be used priority.
     """
 
     velocity_limit: dict[str, float] | float | None = None
     """Velocity limit of the joints in the group. Defaults to None.
 
-    If None, the limit is set to the value specified in the USD joint prim.
+    This limit is used by the actuator model. If None, the limit is set to the value specified in the USD joint prim.
+
+    .. note::
+
+        velocity_limit is not used in ActuatorBaseCfg but is provided for inherited version like
+        :class:`isaaclab.actuators.DCMotor`.
+
+    For ImplicitActuators this value will be collapsed with velocity_limit_sim due to duplicating functionality. If both
+    are set the effort_limit_sim will be used priority.
+    """
```

Comment:
NIT lets make it a note

```suggestion
    .. note::
        For ImplicitActuators this value will be collapsed with velocity_limit_sim due to duplicating functionality. If both
        are set the effort_limit_sim will be used priority.
    """
```